misc/review/IMidiEndpointConnectionSettings.idl

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+// ============================================================================
+// This is part of the Windows MIDI Services App API and should be used
+// in your Windows application via an official binary distribution.
+// Further information: https://github.com/microsoft/MIDI/
+// ============================================================================
+
+#include "namespace_defines.h"
+#include "midl_defines.h"
+MIDI_IDL_IMPORT
+
+namespace MIDI_MIDL_NAMESPACE
```

Comment:
What namespace is all of this in? Do the types mix with 1.0, or is 2.0 in a separate namespace?

Reply:
> What namespace is all of this in? Do the types mix with 1.0, or is 2.0 in a separate namespace?

Separate namespace from MIDI 1.0. After some discussion, Anders recommended we ship the WinRT portion out of box so it's Microsoft.Devices.Midi2 (there's a macro that sets that because we were going back and forth between that and Windows.Devices.Midi2)

Reply:
MIDI 2.0 data format is different, but there are translation functions as part of this SDK to help people move to the new standard. Those function take MIDI 1.0 types or raw data and convert to MIDI 2.0/ (actually MIDI 1.0 protocol in UMP) equivalents.

Reply:
> ship the WinRT portion out of box

Where/how will it ship? The guideline for out-of-box is "Microsoft.Windows", so `Microsoft.Windows.Devices.Midi2`

Reply:
"How will it ship": this change came through just over a week ago. We're sorting out some ship details
- Windows Service and USB MIDI 2.0 class driver: in-box in Windows.
- WinRT SDK: NuGet and GitHub. We may have a separate runtime installer like WinAppSDK so we can centrally service the implementation. We're still working through that.
- Settings and console app: Through the Microsoft Store